examples/Cargo.toml

```diff
@@ -49,7 +49,7 @@ futures = { workspace = true }
 git-version = { workspace = true }
 json5 = { workspace = true }
 log = { workspace = true }
-zenoh = { workspace = true }
+zenoh = { workspace = true, default-features = true }
 zenoh-ext = { workspace = true }
```

Comment:
What about enabling `default-features` as well in `zenoh-ext`?

Reply:
A `cargo tree -f "{p} {f}" ` running under _zenoh/examples_ shows 

```txt
└── zenoh-ext v0.11.0-dev (/home/circle/Workings/ZettaScale/src/zenoh/zenoh-ext) default
    ├── bincode v1.3.3 
    │   └── serde v1.0.188 alloc,default,derive,serde_derive,std (*)
    ├── env_logger v0.11.2 auto-color,color,default,humantime,regex (*)
    ├── flume v0.11.0 async,default,eventual-fairness,futures-core,futures-sink,nanorand,select (*)
    ├── futures v0.3.28 alloc,async-await,default,executor,futures-executor,std (*)
    ├── log v0.4.20 kv_unstable,std,value-bag (*)
    ├── serde v1.0.188 alloc,default,derive,serde_derive,std (*)
    ├── tokio v1.36.0 bytes,default,fs,io-std,io-util,libc,macros,mio,net,num_cpus,rt,rt-multi-thread,socket2,sync,time,tokio-macros (*)
    ├── zenoh v0.11.0-dev (/home/circle/Workings/ZettaScale/src/zenoh/zenoh) auth_pubkey,auth_usrpwd,default,transport_compression,transport_multilink,transport_quic,transport_tcp,transport_tls,transport_udp,transport_unixsock-stream,transport_ws,unstable (*)
    ├── zenoh-core v0.11.0-dev (/home/circle/Workings/ZettaScale/src/zenoh/commons/zenoh-core) default,std (*)
    ├── zenoh-macros v0.11.0-dev (proc-macro) (/home/circle/Workings/ZettaScale/src/zenoh/commons/zenoh-macros)  (*)
    ├── zenoh-result v0.11.0-dev (/home/circle/Workings/ZettaScale/src/zenoh/commons/zenoh-result) default,std (*)
    ├── zenoh-runtime v0.11.0-dev (/home/circle/Workings/ZettaScale/src/zenoh/commons/zenoh-runtime)  (*)
    ├── zenoh-sync v0.11.0-dev (/home/circle/Workings/ZettaScale/src/zenoh/commons/zenoh-sync)  (*)
    └── zenoh-util v0.11.0-dev (/home/circle/Workings/ZettaScale/src/zenoh/commons/zenoh-util) default,std (*)
```

So I think we can omit it.